### PR TITLE
Feature/filter by distance

### LINKED ISSUE #36: Link distance dropdown menu to getMarket/utils cleaning file
Need to add a parameter to one of the fns in utils to make distance dynamic:
(currently hardcoded to 10(miles); line 13 of utils.ts)

### PR DESCRIPTION
**Pull request by AshleyOh-bit**
### Acceptance Criteria
Added drop down menu to filter by distance. Drop down menu has three choices: 15, 25, 50 miles. Linked inputs from select/options to function calls for getMarkets in all instances. Bulk of logic previously written in cleaning function in utils. Refactored input field for numbers, and then resolved merge conflicts with regex zip validation previously added. 
Still need extraneous error handling: telling user what they SHOULD type to input a zip.

### Issues Resolved
#36 #23 #22 

### Code Reviewers
@clairefields15 @Meekb @alexmkio

### CODE REVIEW

File: src/components/Search/Search.tsx [hunk 1]
```diff
@@ -24,8 +25,9 @@ export const Search: React.FC<getMarkets> = ({ getMarkets }) => {
     e: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ) => {
     e.preventDefault();
-    getMarkets(zip);
+    getMarkets(zip, distance);
     setZip('');
+    setDistance(0);
```
Comment (clairefields15) [inline src/components/Search/Search.tsx:30]: Why are we initializing distance to 15 but then resetting it to 0?

**Review by clairefields15 — APPROVED** [review]
Looks great, only the one comment

**Review by Meekb — APPROVED** [review]
TEAM WORK!

File: src/components/Search/Search.tsx (hunk 1 above)
Comment (AshleyOh-bit) [reply src/components/Search/Search.tsx:30]: Good catch! I made a note of this, then totally forgot. Thanks!

**Review by alexmkio — APPROVED** [review]

**Pull request merged**